### PR TITLE
gofumpt to reduce total number of changes versus v5

### PR DESCRIPTION
**Pull request by faddat**
## Description

This is just a formatting change that will ease the review of v5.

This PR replaces #1717 

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [x] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [x] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

File: modules/apps/29-fee/ibc_middleware.go [hunk 1]
```diff
@@ -248,7 +248,7 @@ func (im IBCMiddleware) OnAcknowledgementPacket(
 		return im.app.OnAcknowledgementPacket(ctx, packet, acknowledgement, relayer)
 	}
 
-	var ack = &types.IncentivizedAcknowledgement{}
+	ack := &types.IncentivizedAcknowledgement{}
```
Comment (crodriguezvega) [inline modules/apps/29-fee/ibc_middleware.go:251]: Could you please make here the same modification that @damiannolan [requested in the previous PR](https://github.com/cosmos/ibc-go/pull/1717#discussion_r923161753)?

**Review by crodriguezvega — CHANGES_REQUESTED** [review]
Looks good to me! But just out of curiosity: what was wrong with [the previous PR](https://github.com/cosmos/ibc-go/pull/1717) that you needed to open this new one?

File: modules/apps/29-fee/ibc_middleware.go (hunk 1 above)
Comment (faddat) [reply modules/apps/29-fee/ibc_middleware.go:251]: happy to!  Will neovim it up. 

Would you mind explaining to me the difference?

Comment (faddat) [reply modules/apps/29-fee/ibc_middleware.go:251]: ah-- these are different

Screen Shot 2022-07-21 at 7 58 08 AM

Here it is:
```go
var ack = &types.IncentivizedAcknowledgement{}
ack := &types.IncentivizedAcknowledgement{}
```

There it was:

```go
var ack = &types.IncentivizedAcknowledgement{}
ack := &types.IncentivizedAcknowledgement
```

I don't mind making the change at all, and am instead curious if these are differences in **style** or differences in **function**.

Thanks!

Comment (faddat) [reply modules/apps/29-fee/ibc_middleware.go]: hehehe no I totally understand.

I should close this PR, and go back over to the linting one.

Comment (faddat) [reply modules/apps/29-fee/ibc_middleware.go]: ohhhhhh

hence you included the second line.  THanks!

**Comment by faddat** [comment]
> Looks good to me! But just out of curiosity: what was wrong with https://github.com/cosmos/ibc-go/pull/1717 that you needed to open this new one?

It was accumulating cruft from merging main in.

**Comment by faddat** [comment]
I think it's set.

**Comment by crodriguezvega** [comment]
Do we still need this PR after we merge #1418? Or can this be closed then?

**Comment by faddat** [comment]
Either way really.  You could merge this to make a final review of v5 easier, or merge v5 and you’d still get this.

File: .gitpod.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# This configuration file was automatically generated by Gitpod.
```
Comment (colin-axner) [inline .gitpod.yml:1]: Can you help me understand why this is useful? I'm not familiar with Gitpod

**Review by colin-axner — COMMENTED** [review]
LGTM, just curious on why we should add gitpod?

File: .gitpod.yml (hunk 2 above)
Comment (faddat) [reply .gitpod.yml:1]: oh no I am sorry!

Comment (faddat) [reply .gitpod.yml:1]: OK, so 1 bad thing about gitpod:

It puts these in repos automatically.  

So I guess not seeing them in unrelated PR's could be a good reason to do it, but that's not really a good reason.  Let me get into an actual good reason, after deleting this file.

**Review by colin-axner — APPROVED** [review]

**Review by damiannolan — APPROVED** [review]
LGTM 🚀

**Pull request merged**